plugins/UsersManager/templates/userSettings.twig

```diff
@@ -120,6 +120,20 @@
     </form>
 </div>
 
+{% if showNewsletterSignup %}
+<div piwik-content-block id="newsletterSignup" content-title="{{ 'UsersManager_NewsletterSignupTitle'|translate|e('html_attr') }}">
+
+    <div id="newsletterSignupMsg">{{ 'UsersManager_NewsletterSignupMessage'|translate|raw }}<br/><br/></div>
+    <div id="newsletterSignupFailure" style="display:none">{{ 'UsersManager_NewsletterSignupFailureMessage'|translate|raw }}<br/><br/></div>
+    <div id="newsletterSignupSuccess" style="display:none">{{ 'UsersManager_NewsletterSignupSuccessMessage'|translate|raw }}<br/><br/></div>
+
+    <button class="btn btn-link" id="newsletterSignupBtn"
+            ng-controller="PersonalSettingsController as personalSettings"
+            ng-click="personalSettings.signupForNewsletter()"
+    >{{ 'General_Yes'|translate }}</button>
+</div>
+{% endif %}
```

Comment:
@mattab Should there be any specific copy in this new section? The screenshot makes the section look a bit bare.